public/components/game/details/details_test.js

```diff
@@ -13,6 +13,7 @@ var DetailsViewModel = details.ViewModel;
 QUnit.module("bitballs/game/details/", {
     setup: function() {
         localStorage.clear();
+        fixture.delay = 1;
```

Comment:
No. It's actually the other way around. With the delay defined the tests were breaking. 

I spent the afternoon yesterday attempting to recreate the breaking test locally (on my Ubuntu VM), which worked well when I was originally debugging the issue. Unfortunately I haven't been able to recreate the failure since.